web/src/main/resources/application.properties.EXAMPLE

```diff
@@ -11,6 +11,10 @@ genexrefs.url=http://grch37.rest.ensembl.org/xrefs/id/ACCESSION?content-type=app
 #"VARIANT" will be replaced with actual variant upon each request.
 mutationAssessor.url=http://mutationassessor.org/r3/?cm=var&var=VARIANT&frm=json&fts=input,rgaa,rgvt,msa,pdb,F_impact,F_score,vc_score,vs_score,info,var,gene,uprot,rsprot,gaps,msa_height,chr,rs_pos,rs_res,up_pos,up_res,cnt_cosmic,cnt_snps
 
+# my variant info web API URL
+#"VARIANT" will be replaced with actual variant upon each request.
+myVariantInfo.url=http://myvariant.info/v1/variant/VARIANT
```

Comment:
maybe better to have this all lower case - i see that mutationAssessor.url also has camelcase, but that was an oversight i think. Better to only have lower case properties imo